### DISCUSSION TITLE
What's wrong? How to fix it.

### DISCUSSION THREAD

**Discussion by moresol** [LazyVim/LazyVim#2065] [Q&A]
![Screenshot 2023-11-25 at 8 51 20 PM](https://github.com/LazyVim/LazyVim/assets/106124331/db90cbfd-aab4-4b53-b035-00946265b2cb)

**Comment by dpetka2001** [comment]
As the error says you have made changes to `~/.local/share/nvim/lazy/bufferline.nvim`. You should never do that, because it causes conflicts when trying to update the plugin from its repo. Delete the directory and restart Neovim, so the plugin can be installed again.

**Reply by moresol** [reply]
Sorry I'm so new to this thing, should do it like this? And then just run shell command nvim ? rm -rf ~/.local/share/nvim/lazy/bufferline.nvim

**Reply by moresol** [reply]
I was just hanging around exploring things in iterm2, nothing special, then boom got the error what I posted

**Reply by dpetka2001** [reply]
You must have made some changes to the files in that directory. If that is the only directory you get errors for, then just delete that. If you have more directories with the same error, then just delete the whole `~/.local/share/nvim/lazy` directory and restart Neovim so that everything gets reinstalled. And for future reference, just make sure you don't do any changes in that directory.

**Reply by moresol** [reply]
I'm gonna try, thank you, I'll let you know if something go wrong.

**Reply by moresol** [reply]
Genius, you are a lifesaver 🤩. That helped "then just delete the whole ~/.local/share/nvim/lazy"
